test/helpers/browser-env.js

```diff
@@ -0,0 +1,22 @@
+import { DOMParser } from 'xmldom'
+import { jsdom } from 'jsdom'
+import { File } from 'file-api'
+
+
+global.document = jsdom('<body></body>')
+global.DOMParser = DOMParser
+
+File.prototype.slice = function (start = 0, end = null, contentType = '') {
```

Comment:
We can simulate environment for unsupported browsers. Depending on the depth you'd want to go to, we could simulate all sorts of different browser environments. I'd start with simply not having File/Blob (as you test in Evaporate), then go with document, window, DOMParser and so on. Maybe we can test for proper `slice` too, however, there is not really a good `file-api` implamentation for Node yet.